docbook/reference/en/en-US/modules/Reactive.xml

```diff
@@ -1,63 +1,1263 @@
+                https://html.spec.whatwg.org/multipage/server-sent-events.html</ulink> for details.)
+                In particular, 1) SSE events are meant to hold text data, represented in character set UTF-8. In the general streaming mode,
+                certain delimiting characters in the data ('\r', '\n', and '\') are escaped so that arbitrary binary data can be
+                transmitted. Also, 2) the SSE specification requires the client to reconnect if it gets disconnected. If the stream
+                is finite, reconnecting will induce a repeat of the stream, so SSE is really meant for unlimited streams.
+                In general streaming mode, the client will close, rather than automatically reconnect, at the end of the stream. It follows
+                that this mode is suitable for finite streams.
+                
+                <para>
+                   <emphasis role="bold">Note. </emphasis> The Content-Type header in general streaming mode is set to
+                </para>
+                
+                <programlisting>
+          applicaton/x-stream-general;"element-type=&lt;element-type&gt;"
+                </programlisting>
+                <para>
+                   where &lt;element-type&gt;
+                   is the media type of the data elements in the stream. The element media type is derived
+                   from the @Produces annotation. For example, 
+                </para>
+      
+                <programlisting>
+      @GET
+      @Path("flowable/thing")
+      @Stream
+      @Produces("application/json")
+      public Flowable&lt;Thing&gt; getFlowable() { ... }
+                </programlisting>
+                
+                <para>
+                   induces the media type
+                </para>
+                
+                <programlisting>
+          application/x-stream-general;"element-type=application/json"
+                </programlisting>
+                
+                <para>
+                   which describes a stream of JSON elements.
+                </para>
+            </listitem>
+        </varlistentry>
+        <varlistentry>
+            <term><code>@Stream(Stream.MODE.RAW)</code></term>
+            <listitem>
+                In this case each value is written directly to the wire, without any formatting, as it becomes available. 
+                This is most useful for values that can be cut in pieces, such as strings, bytes, buffers, etc., and then
+                re-concatenated on the client side. Note that without delimiters as in
+                general mode, it isn't possible to reconstruct something like <classname>List&lt;List&lt;String&gt;&gt;</classname>.
+
+                <para>
+                   <emphasis role="bold">Note. </emphasis> The Content-Type header in raw streaming mode is set to
+                </para>
+
+                <programlisting>
+          applicaton/x-stream-raw;"element-type=&lt;element-type&gt;"

```

Comment:
I'm sorry I only noticed this now @ronsigal but I think it's wrong: in all use-cases I can think of, I need the content type to be set by `@Produces` and not set to the useless `x-stream-raw` that no existing client can handle. Can we change this behaviour?